src/test/kotlin/io/snyk/plugin/UtilsKtTest.kt

```diff
@@ -0,0 +1,77 @@
+package io.snyk.plugin
+
+import com.intellij.openapi.vfs.VirtualFile
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.unmockkAll
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertFalse
+import junit.framework.TestCase.assertTrue
+import org.apache.commons.lang3.SystemProperties
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+class UtilsKtTest {
+
+    @Before
+    fun setUp() {
+        unmockkAll()
+        mockkStatic(SystemProperties::class)
+        every { SystemProperties.getOsName() } returns "Windows"
+    }
+
+    @After
+    fun tearDown() {
+        unmockkAll()
+    }
+
+    @Test
+    fun toLanguageServerURL() {
+        val path = "C:/Users/username/file.txt"
+        var uri = "file://$path"
+        var virtualFile = mockk<VirtualFile>()
+        every { virtualFile.url } returns uri
+
+        assertEquals("file:///$path", virtualFile.toLanguageServerURL())
+
+        uri = "file://$path"
+        virtualFile = mockk<VirtualFile>()
+        every { virtualFile.url } returns uri
+
+        assertEquals("file:///$path", virtualFile.toLanguageServerURL())
```

Comment:
I might be missing something but is there a difference between this test and the one before? Did you mean to set `uri` to `file:///$path` instead?

Reply:
Yeah, a slash is missing 🙈